components/email/src/Email.svelte

```diff
@@ -1027,18 +1092,35 @@
                     </div>
                   {:else}
                     <div class="message-head">
-                      <div class="message-from">
-                        <span class="name"
-                          >{message.from[0].name || message.from[0].email}</span
-                        >
-                        <!-- tooltip component -->
-                        <nylas-tooltip
-                          on:toggleTooltip={setTooltip}
-                          id={message.id.slice(0, 3)}
-                          {current_tooltip_id}
-                          icon={DropdownSymbol}
-                          content={message.from[0].email}
-                        />
+                      <div class="avatar-from">
+                        {#if show_contact_avatar}
+                          <div
+                            class="default-avatar"
+                            data-email={message.from[0].email}
+                            data-contact={JSON.stringify(
+                              contacts[message.from[0].email],
+                            )}
+                          >
```

Comment:
Not sure why the `data-contact` and `data-email` attributes have been added here? Do these elements need the contact object?
<img width="756" alt="Screen Shot 2021-09-01 at 12 16 30 PM" src="https://user-images.githubusercontent.com/55773810/131706844-53b918db-61f8-4e72-8b6f-57fdb204c97d.png">

Reply:
Oh that was for debugging, it should definitely go. Thank you for pointing it out!